ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -1773,6 +1773,26 @@
                 "public record IfrsVariable : BaseDataRecord",
                 "\n{",
                 "\n    public double Value { get; init; }",
+                "\n    public double Value1 { get; init; }",
+                "\n    public double Value2 { get; init; }",
+                "\n    public double Value3 { get; init; }",
+                "\n    public double Value4 { get; init; }",
+                "\n    public double Value5 { get; init; }",
+                "\n    public double Value6 { get; init; }",
+                "\n    public double Value7 { get; init; }",
+                "\n    public double Value8 { get; init; }",
+                "\n    public double Value9 { get; init; }",
+                "\n    public double Value10 { get; init; }",
+                "\n    public double Value11 { get; init; }",
+                "\n    public double Value12 { get; init; }",
+                "\n    public double Value13 { get; init; }",
+                "\n    public double Value14 { get; init; }",
+                "\n    public double Value15 { get; init; }",
+                "\n    public double Value16 { get; init; }",
+                "\n    public double Value17 { get; init; }",
+                "\n    public double Value18 { get; init; }",
+                "\n    public double Value19 { get; init; }",
```

Comment:
Do we really need 19 extra Value?

Reply:
It is important not to have more property than those we need. So it is important to know which are actually needed. Consistency with ProjectionConfiguration values is not the right criteria.

Reply:
Ok, then I remove the IfrsVariable's 19th Value from the DataStructure.

Reply:
Done.